packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -818,11 +821,54 @@ nodeunitShim({
 
     test.done();
   },
-});
 
+  'when cluster identifier has uppercase letters (when the clusterIdentifierToLowercase feature flag is enabled)'(test: Test) {
```

Comment:
This is kind of a weird test description 😜. How about something like

```suggestion
  'changes the case of the cluster identifier if the clusterIdentifierToLowercase feature flag is enabled'(test: Test) {
```